tests/examples/opinions/united_states/mass_example.compare.json

```diff
@@ -1,52 +1,446 @@
 [
   {
-    "case_dates": "2024-07-03",
-    "case_names": "Commonwealth v. O'Brien",
-    "download_urls": "/doc/comm-v-obrien-13078/download",
+    "case_dates": "2024-11-19",
+    "case_names": "ROBERT M. JOOST v. BOARD OF BAR EXAMINERS",
+    "download_urls": "https://www.socialaw.com/services/slip-opinions/slip-opinion-details/robert-m.-joost-vs.-board-of-bar-examiners",
     "precedential_statuses": "Published",
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
-    "docket_numbers": "SJC 13078",
-    "case_name_shorts": "Commonwealth"
+    "docket_numbers": "SJC-13587",
+    "judges": "",
+    "summaries": "The petitioner, Robert M. Joost, appeals from the judgment of a single justice of this court denying without a hearing his petition to waive certain educational requirements imposed by S.J.C. Rule 3:01, as appearing in 478 Mass. 1301 (2018), for admission to the bar of the Commonwealth. Specifically, Joost seeks waiver of the requirements under S.J.C. Rule 3:01 that petitioners hold a bachelor's degree or the equivalent and a juris",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2024-11-14",
+    "case_names": "TRUSTEES OF BOSTON UNIVERSITY & others[1] v. CLERK-MAGISTRATE OF THE CAMBRIDGE DIVISION OF THE DISTRICT COURT DEPARTMENT & others [2]",
```

Comment:
should the case name be cleaned of the square brackets?

Reply:
those are footnotes - and yes we should remove them from here and in the judges too.  they will remain untouched in the source HTML